classes/core-image-pelux.bbclass

```diff
@@ -24,6 +24,9 @@ IMAGE_INSTALL_append = "\
     dlt-daemon         \
     dlt-daemon-systemd \
     node-state-manager \
+    portaudio-v19      \
```

Comment:
portaudio is only needed for a test app, right?

also, why is this in GENIVI section?

Reply:
Yes, I am 99.99% sure. :) 

@TSAnsari Will there be a recipe for the test application? Is it not enough to add this dependency there? If there will not be a recipe, there must be some instructions for how to build the test app? If so, I'd say just add something like "bitbake portaudio-v19 and install the rpm:s" there. But... if there will be a recipe I would assume adding portaudio-v19 as a dependency in it would make this redundant.

Reply:
Hum, does the same go for the other dependencies as well? Should work by just having them as dependencies in relevant recipes? Does that make sense?

Reply:
> portaudio is only needed for a test app, right?
> 
> also, why is this in GENIVI section?

yes its only used by the sample application. Sure I will put it outside the GENIVI section.

Reply:
> portaudio is only needed for a test app, right?
> 
> also, why is this in GENIVI section?

So my thought was like this,
Since Sample application is part of the avs-device-sdk and is built along with it, we should install it along with avs-device-sdk. This is a temporary thing and should be removed once we develop our own alexa application/service. Then there will be separate recipe for our own application and depending on if we use portaudio or not, we can keep or remove it.

Reply:
> Yes, I am 99.99% sure. :)
> 
> @TSAnsari Will there be a recipe for the test application? Is it not enough to add this dependency there? If there will not be a recipe, there must be some instructions for how to build the test app? If so, I'd say just add something like "bitbake portaudio-v19 and install the rpm:s" there. But... if there will be a recipe I would assume adding portaudio-v19 as a dependency in it would make this redundant.


So my thought was like this,
Since Sample application is part of the avs-device-sdk and is built along with it, we should install it along with avs-device-sdk. This is a temporary thing and should be removed once we develop our own alexa application/service. Then there will be separate recipe for our own application and depending on if we use portaudio or not, we can keep or remove it.

Reply:
I think we should package sampleapp separately then

Reply:
> I think we should package sampleapp separately then

Sample application is part of the sdk and is build along with the sdk, so that is the reason to add it here. We can add our own alexa service in separate recipe when we create one.

Reply:
nothing stops you from packaging it separately

Reply:
Yes, but it feels unnecessary to create a separate recipe for sample app only to throw it away later. I remember we discussed this during standup and you also agreed on this. May be we can discuss this again tomorrow about what would be suitable approach.

Also this commit is not related to sample app. This commit only adds alexa sdks and portaudio to the image.

May be I should not add portaudio in this commit and use a different commit(along with commit for Sample app)

Reply:
> ...it feels unnecessary to create a separate recipe ...
a package, not a recipe.

Reply:
> May be I should not add portaudio
Yes, you shouldn't. Only sample app should depend on it.

Reply:
> > ...it feels unnecessary to create a separate recipe ...
> 
> a package, not a recipe.

Here is the PR that adds sample app
https://github.com/Pelagicore/meta-bistro/pull/219
Have a look and comment.

For this commit I will remove portaudio

Reply:
> > May be I should not add portaudio
> 
> No, you shouldn't. Only sample app should depend on it.

portaudio removed

Reply:
> > > May be I should not add portaudio
> > 
> > 
> > No, you shouldn't. Only sample app should depend on it.
> 
> portaudio removed

Here